presto-main/src/main/java/io/prestosql/operator/PageUtils.java

```diff
@@ -30,37 +30,30 @@ public static Page recordMaterializedBytes(Page page, LongConsumer sizeInBytesCo
         // account processed bytes from lazy blocks only when they are loaded
         Block[] blocks = new Block[page.getChannelCount()];
         long loadedBlocksSizeInBytes = 0;
-        boolean allBlocksNonLazy = true;
+        boolean allBlocksLoaded = true;
 
         for (int i = 0; i < page.getChannelCount(); ++i) {
             Block block = page.getBlock(i);
-            if (block instanceof LazyBlock) {
-                LazyBlock delegateLazyBlock = (LazyBlock) block;
-                if (delegateLazyBlock.isLoaded()) {
```

Comment:
Looks like we lost this case, which seems fine to me.  If this is a behavior change maybe move it to a separate commit, got keep this one more mechanical.